doc/sphinx-guides/source/qa/other-approaches.md

```diff
@@ -0,0 +1,121 @@
+
+1. Validate the documentation.
+
+    Build the doc using Jenkins, does it build without errors?
+    Read it through for sense.
+    Use it for test cases and to understand the feature.
+
+1. Build and deploy the pull request.
+
+    Normally this is done using Jenkins and automatically deployed to the QA test machine.
+
+1. Configure if required
+
+    If needed to operate and everyone installing or upgrading will use this, configure now as all testing will use it.
+
+1. Smoke test the branch.
+    
+    Standard, minimal test of core functionality.
+
+1. Regression test-related or potentially affected features
+
+    If config is optional and testing without config turned on, do some spot checks/ regression tests of related or potentially affected areas. 
+
+1. Configure if optional
+
+    What is the default, enabled or disabled? Is that clearly indicated? Test both.
+    By config here we mean enabling the functionality versus choosing a particular config option. Some complex features have config options in addition to enabling. Those will also need to be tested.
+
+1. Test all the new or changed functionality.
+
+    The heart of the PR, what is this PR adding or fixing? Is it all there and working?
+
+1. Regression test related or potentially affected features.
+    
+    Sometimes new stuff modifies and extends other functionality or functionality that is shared with other aspects of the system, e.g. export, import. Check the underlying functionality that was also modified but in a spot check or briefer manner.
+
+1. Report any issues found within the PR
+
+    It can be easy to lose track of what you’ve found, steps to reproduce, and any errors or stack traces from the server log. Add these in a numbered list to a comment in the pr. Easier to check off when fixed and to work on. Add large amounts of text as in the server log as attached, meaningfully named files.
+
+1. Retest all fixes, spot check feature functionality, smoke test
+    
+    Similar to your initial testing, it is only narrower.
+
+1. Test upgrade instructions, if required
+
+    Some features build upon the existing architecture but require modifications, such as adding a new column to the DB or changing or adding data. It is crucial that this works properly for our 100+ installations. This testing should be performed at the least on the prior version with basic data objects (collection, dataset, files) and any other data that will be updated by this feature. Using the sample data from the prior version would be good or deploying to dataverse-internal and upgrading there would be a good test. Remember to back up your DB before doing a transformative upgrade so that you can repeat it later if you find a bug.
+
+1. Make sure the API tests in the PR have been completed and passed.
+   
+    They are run with each commit to the PR and take approximately 42 minutes to run.
+
+1. Merge PR
+    
+    Click merge to include this PR into the common develop branch.
```

Comment:
There are three merge options. Should we be more specific about which one we use (and maybe why?):

![Screenshot 2023-11-14 at 8 13 24 AM](https://github.com/IQSS/dataverse/assets/21006/4d29333e-1c31-4509-b67c-630eb36dfab7)